.github/workflows/repo.yaml

```diff
@@ -19,15 +19,22 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: yarn build
   test:
+    strategy:
+      matrix:
+        shard: [1/4, 2/4, 3/4, 4/4]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version: 18
           cache: "yarn"
+      - uses: actions/cache@v3
+        with:
+          path: /tmp/jest_rt
+          key: ${{ hashFiles('**/yarn.lock') }}-${{ matrix.shard }}
       - run: yarn install --frozen-lockfile
-      - run: yarn test
+      - run: yarn test:withFlags --maxWorkers=2 --shard=${{ matrix.shard }} --passWithNoTests
```

Comment:
Would there be an issue using the double-dash here rather than in the new command?  E.g.:
```
yarn test -- --maxWorkers=2 --shard=${{ matrix.shard }} --passWithNoTests
```

Truly no issue with it as-is, just more used to doing something like that in these kinds of situations.

Reply:
It didn't want to pass through the arguments correctly when I tried that^^

Lerna tries to interpret the `maxWorkers` argument

Reply:
Ah, fun!